.github/CONTRIBUTING.md

```diff
@@ -48,22 +48,22 @@ No problem! You can learn how from this *free* series, [How to Contribute to an
 Alternatively, ask us for help.
 
 # Open Development
-All work on Amido Stacks is on GitHub. Both Amido maintainers and external contributors send pull requests which go through the same review process. We are using [Azure Pipelines](https://azure.microsoft.com/en-gb/services/devops/pipelines/) as our CI/CD tool of choice. Every PR will invoke a series of steps to build, test, and deploy.
+All work on Ensono Stacks is on GitHub. Both Ensono maintainers and external contributors send pull requests which go through the same review process. We are using [Azure Pipelines](https://azure.microsoft.com/en-gb/services/devops/pipelines/) as our CI/CD tool of choice. Every PR will invoke a series of steps to build, test, and deploy.
 
 ## Code changes happen through pull requests
-We use Github for hosting and supporting our open source Stack projects. The Amido internal maintainers team will be monitoring Github for pull requests (PRs) and issues.
+We use Github for hosting and supporting our open source Stack projects. The Ensono internal maintainers team will be monitoring Github for pull requests (PRs) and issues.
 
 We actively welcome your pull requests:
 
 1. Fork the repo and create your branch from master
 2. If you've added code that should be tested, add tests.
 3. Update the documentation.
 4. Ensure the test suite passes.
-5. Issue the pull request using the repos PULL_REQUEST_TEMPLATE.md. It's important to link the [issue number](https://github.com/amido/amido.github.io/issues) to ensure that the Issue is closed.
+5. Issue the pull request using the repos PULL_REQUEST_TEMPLATE.md. It's important to link the [issue number](https://github.com/Ensono/amido.github.io/issues) to ensure that the Issue is closed.
 
 ## How to report a bug
 ### Security first
-If you find a security vulnerability, do NOT open an [issue](https://github.com/amido/amido.github.io/issues/new/choose). Email [Amido](mailto:stacks@amido.com) instead.
+If you find a security vulnerability, do NOT open an [issue](https://github.com/Ensono/amido.github.io/issues/new/choose). Email [Ensono](mailto:stacks@amido.com) instead.
```

Comment:
Do we have an ensono email for this `stacks@amido.com`?

Reply:
Yes we do, updated.